### DISCUSSION TITLE
하나은행 체험형 인턴 면접 후기

### DISCUSSION THREAD

**Discussion by 1jeongg** [BDD-CLUB/challenge-every-week-posting#604] [5. 회고]
이번 주 금요일에 갔다 온 따끈 따끈한 후기입니다,, 결과는 7월 회고에..!

요즘 공부는 안하고 회고와 후기로 가득 채우네요.. 다음 주부터는 졸과 프로젝트 하는 내용도 쓰고 스프링 공부도.. 다시.... ㅠㅠ

[2024 하나은행 체험형 인턴(디지털/ICT) 지원 후기](https://1jeongg.github.io/hana-intern-interview/)

여담으로 서울 너무 더워서 깜짝 놀랐습니다.

**Comment by 02ggang9** [comment]
재미있게 잘 봤습니다~~ 👍👍

**Comment by stopmin** [comment]
와~~~ 금융권 인턴!!! 저도 꼭 면접 한번 봐보고 싶네요 ㅎㅎㅎ
